serial/serialutil.py

```diff
@@ -649,19 +649,19 @@ def read_all(self):
         """
         return self.read(self.in_waiting)
 
-    def read_until(self, terminator=LF, size=None):
+    def read_until(self, expected=LF, size=None):
```

Comment:
renaming the parameter may break some code, but it makes sense to align the name with the function from telnetlib

Reply:
Confirming that this broke some code. Does this project not use semantic versioning? This rolled out with version 3.5b0 if I follow the history correctly, yet it included incompatible API changes such as this. The changelog lumps this in with "doc updates" but changing a parameter name is not only for documentation.